recipes/pyjulia/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.4" %}
+{% set name = "julia" %}
+
+package:
+  name: pyjulia
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea83e23a35a4a60014654cbcab70daf1da7539dc2129a19bb7e6354a168bad70
+
+build:
+  skip: True  # [osx or win]
+  number: 0
+  script: python -m pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - julia
+  run:
+    - python
+    - julia
+
+test:
+  commands:
+    - python -c 'import julia; julia.Julia()'
+
+about:
+  home: https://github.com/JuliaPy/pyjulia
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  summary: "python interface to julia"
+
+extra:
+  recipe-maintainers:
+    - Carreau
+    - ViralBShah
```

Comment:
Are you ok being listed as a maintainer on this recipe, @ViralBShah?

Reply:
I doubt I really can contribute in any meaningful way directly, but will be happy to help in any way I can.

Reply:
There's honestly not too much required from individual maintainers. Here is a [description]( https://github.com/conda-forge/staged-recipes#9-what-does-being-a-conda-forge-feedstock-maintainer-entail ).